Business.Queries/Dtos/JobGroupDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Business.Queries.Dtos
```

Comment:
All the DTO's should be in the Business.Dtos project. This allows them to be used in both the Business and Web projects.